app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: type | Cafe) -> str:
```

Comment:
In Python, it's more conventional to use the `isinstance` function to check if an object is of a certain type. Here you're using the `type | Cafe` notation, which is not very common in Python. Consider refactoring this to use `isinstance` inside the function body. This is a minor issue and doesn't affect the functionality of your code, but it's a good practice to follow the conventions of the language you're using.